package.json

```diff
@@ -78,6 +78,7 @@
     "jquery-touch-events": "git+https://github.com/benmajor/jQuery-Touch-Events.git",
     "jquery.payment": "^3.0.0",
     "knox": "^0.9.2",
+    "lodash.get": "^4.4.2",
```

Comment:
I'm just randomly dropping by 👋 , but I think you're looking for http://underscorejs.org/#result